components/style/values/computed/length.rs

```diff
@@ -70,7 +72,8 @@ impl CalcLengthOrPercentage {
     #[inline]
     #[allow(missing_docs)]
     pub fn length(&self) -> Au {
-        self.length
+        assert!(self.percentage.is_none());
```

Comment:
I had to remove it altogether, given we also use this method from `LengthOrPercentage`.